src/pages/Bid/BidPerchase.jsx

```diff
@@ -0,0 +1,276 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import SuggestPricePerchaseComponent from './SuggestPricePerchaseComponent';
+import ImmediatePerchaseComponent from './ImmediatePerchaseComponent';
+import BidDeadlineComponent from './BidDeadlineComponent';
```

Comment:
- 스타일 관련 import 는 맨 마지막에 해주세요!
```suggestion
import React, { useEffect, useState } from 'react';
import SuggestPricePerchaseComponent from './SuggestPricePerchaseComponent';
import ImmediatePerchaseComponent from './ImmediatePerchaseComponent';
import BidDeadlineComponent from './BidDeadlineComponent';
import styled from 'styled-components';
```